Tests/SwiftDocCTests/Test Bundles/MixedLanguageFrameworkSingleLanguageParent.docc/MixedLanguageFramework.md

```diff
@@ -4,6 +4,6 @@
 
 ### Errors
 
-- ``MyError-swift.struct/Code-swift.enum``
```

Comment:
I tried to preserve previous behaviors as much as possible to avoid breaking existing links.

This change is not an issue since it used to be non-deterministic if it needed the disambiguation suffix or not. Any link that relied on this behavior would break every now and then.

Reply:
Makes sense. When I wrote this test, I wasn't sure if it needed a disambiguation prefix or not—it seems like the answer is no :)